python/plugins/db_manager/layer_preview.py

```diff
@@ -131,3 +131,4 @@ def _loadTablePreview(self, table, limit=False):
             self.currentLayer = vl
 
             self.freeze(False)
+            super().refresh()
```

Comment:
is it necessary to call super().refresh or not just refresh(). In this way if a overloading is necessary it will be executed. Is it the super() method exactly to avoid use of the overloaded?

Reply:
@luipir this is necessary to access inherited methods that have been overridden in LayerPreview class (i.e. ```self.refresh()```). In this case it is necessary to call ```QgsMaoCanvas::refresh()```which is not being called from ```QgsMapCanvas::freeze()```. In previous version we had used ```QgsMapCanvas::setRendereFlag()``` which calls the refresh method.